addons/hr_holidays/static/src/core/common/message_patch.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-inherit="mail.Message" t-inherit-mode="extension">
+        <xpath expr="//img[hasclass('o-mail-Message-avatar')]" position="after">
+            <div class="d-flex justify-content-center flex-shrink-0 position-absolute top-100 start-100 translate-middle mt-n1 ms-n1 align-items-center rounded-circle bg-inherit">
+                <div t-if="message.author.out_of_office_date_end" class="d-flex justify-content-center flex-shrink-0 align-items-center rounded-circle bg-inherit">
+                    <span class="d-flex flex-column">
+                        <i class="fa fa-plane text-warning small" role="img"/>
+                    </span>
+                </div>
+            </div>
+        </xpath>
```

Comment:
Should use `ImStatus` component when there's `leave_` in `message.author.im_status`.
That way the size is consistent with rest, and also it takes into account the green plan when on leave but online.